drivers/dai/intel/ssp/dai-params-intel-ipc4.h

```diff
@@ -223,6 +226,28 @@ struct dai_intel_ipc4_gateway_config_blob {
 /* i2s Configuration BLOB building blocks */
 
 /* i2s registers for i2s Configuration */
+#ifdef CONFIG_SOC_INTEL_ACE30_PTL
+struct ssp_rx_dir {
+	uint64_t ssmidytsa;
+} __packed;
+
+struct ssp_tx_dir {
+	uint64_t ssmodytsa;
+} __packed;
+
+struct dai_intel_ipc4_ssp_config {
+	uint32_t ssc0;
+	uint32_t ssc1;
+	uint32_t sscto;
+	uint32_t sspsp;
+	uint32_t ssc2;
+	uint32_t sspsp2;
+	uint32_t ssc3;
+	uint32_t ssioc;
+	struct ssp_rx_dir rx_dir[I2SIPCMC];
+	struct ssp_tx_dir tx_dir[I2SOPCMC];
```

Comment:
Why copy/paste the whole structure? Again, this makes it tedious to spot differences (for a totally different reason). 

```C
    ...
    uint32_t ssc3;
	uint32_t ssioc;
#ifdef CONFIG_SOC_INTEL_ACE30_PTL
	struct ssp_rx_dir rx_dir[I2SIPCMC];
	struct ssp_tx_dir tx_dir[I2SOPCMC];
#endif
} __packed;
```